network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -64,7 +64,7 @@ The 1.4 release of the Network Observability Operator adds improvements and new
 ** The *NetObserv / Health* dashboard shows flows overhead as well as top flow rates per nodes, namespaces, and workloads.
 ** Infrastructure and Application metrics are shown in a split-view for namespaces and workloads.
 
-For more information, see xref:../network_observability/network-observability-overview.adoc#network-observability-dashboards[Network Observability metrics] and xref:../network_observability/observing-network-traffic.adoc#network-observability-quickfilternw-observe-network-traffic[Quick filters].
+For more information, see xref:../network_observability/metrics-alerts-dashboards.adoc#network-observability-viewing-dashboards_metrics-dashboards-alerts[View metrics dashboards] and xref:../network_observability/observing-network-traffic.adoc#network-observability-quickfilternw-observe-network-traffic[Quick filters].
```

Comment:
The title of the page this links to is called "Viewing Network Observability metrics dashboards". Suggest matching that here